src/components/calcite-pick-list/calcite-pick-list.scss

```diff
@@ -35,3 +35,7 @@ calcite-filter {
 :host([loading][disabled]) {
   min-height: 2rem;
 }
+
+::slotted(calcite-dropdown) {
```

Comment:
Is there a way this can be tackled in a generic way? Having component-specific fixes will not scale well.

@driskull Will setting `position:relative` on the shadow DOM root container improve positioning or does it have to go on the host?

Reply:
Shadow dom root container should work and is preferred 👍